client/src/components/FriendsPollCard.js

```diff
@@ -93,25 +96,33 @@ class FriendsPollCard extends React.Component {
                         <CardActions className={classes.votesContainer}>
                             <div className={classes.votes}>
                                 <IconButton
-                                    disabled={hasVoted}
+                                    disabled={votedForOption === 0}
                                     onClick={() => this.registerVote(0)}
                                     className={classes.icon}
                                     aria-label="Votes for first image"
                                     component="span">
-                                    <Icon>favorite</Icon>
+                                    {votedForOption === 0 ? (
+                                        <Favorite />
```

Comment:
I found out you can use the old way `<Icon>favorite</Icon>` and `<Icon>favorite_border</Icon>` without importing the new package. It will look the same because I'm actually importing material icons in index.html

Reply:
ok I will change it